src/pages/PlacePage.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="test-page">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <div class = "searchbar">
+      <center>
+      <input class="text-search" type="text" v-model="search" placeholder="Nama Tempat Yang dicari" @change="searching" >
+      <input class="button" type="submit" value ="cari">
+      </center>
+    </div>
+    <div class= "box">
+        <div class = "item" v-for="place in places" :key="place.Id">
+        <header>
+            <img :src="place.Company.Photo.Url" class="photo">
+            <div class = "partner">{{ place.Company.Name }}</div>
+            <div class = "non-rate" v-if="place.Rating == null"> </div>
+            <div class = "rate" v-else> <span class="fa fa-star checked"></span>{{ place.Rating }}/{{ place.RatingCount }} </div>
+        </header>
+            <div class="item-body">
+                   <img :src="place.Photos[0].Url">
+               <div class="item-details">
+                   <div class="name">{{ place.Name }} </div>
+                   <div class="city">{{ place.City.Name }} </div>
+                   <div class="hour">{{ moment (place.Times[0].OpenHour).format('LT') }} - {{ moment (place.Times[0].CloseHour).format('LT') }}
+                      <span class="close" v-if="checkHourInfo(place.Times)"> Close Now </span>
```

Comment:
Your implementation to show the operation hours is not completely right.

In here, you only get the first `OpenHour` and `CloseHour` in that `Times` list. What if the first day in that list is `Monday` but today is `Tuesday`? In addition, the operation hours in those days is different, then your implementation goes wrong because that's not the right operation hours for this day

**Example:** 
Monday: 10.00 - 15:00
Tuesday: 15:00 - 17:00 (Today)

You can see a different here isn't it?

Reply:
You need to check `TimeId` in the `Times` list. 

`TimeId` 1 is monday
`TimeId` 2 is tuesday
`TimeId` 3 is wednesday
and so on..

Reply:
Here's an example for this algorithm:

```
currentDate = get today date
currentDay = get day from current date
timeObject = find object in the list Times by checking TimeId must same with this day

currentTime = get current time by moment
openHour = get open hour time by moment from timeObject
closeHour = get close hour time by moment from timeObject

if (currentTime > openHour && now < closeHour) then
  'buka'
else
  'tutup'
```